web/build.gradle

```diff
@@ -61,7 +61,7 @@ build.dependsOn zipResources
 
 gretty {
     def baseUrl = 'http://localhost:8080'
-    def jvmArgSpringProfiles = '-Dspring.profiles.active=dev,caching-enabled,testapi,embedded,hsa-stub,wc-hsa-stub,security-both,noprocessing'
+    def jvmArgSpringProfiles = '-Dspring.profiles.active=dev,caching-enabled,testapi,embedded,hsa-stub,wc-hsa-stub,security-both,noprocessing,ia-stub'
```

Comment:
I devops/openshift/dev/env/secret-env.sh ovan används proflien "it-stub". Har det blivit ett misstag här eller är det meningen att det ska vara olika?

Reply:
Nu är det fixat för openshift, ska vara ia-stub